lib/freeze.js

```diff
@@ -37,42 +34,53 @@ module.exports = class Freeze {
     });
   }
 
-  freeze(context, props) {
+  async freeze(context, props) {
     const labels = context.payload.issue.labels;
     if (GitHubHelper.isLabelOnIssue(
       context.payload.issue,
       this.config.labelName) === undefined) {
       labels.push(this.config.labelName);
     }
 
-    this.github.issues.edit(context.issue({
+    const kv = metadata(context.github, context.issue(), process.env.APP_ID);
+    await kv.set('snooze', props);
+
+    await this.github.issues.edit(context.issue({
       state: 'closed',
       labels
     }));
 
-    this.github.issues.createComment(context.issue({
+    await this.github.issues.createComment(context.issue({
       body: 'Sure thing. I\'ll close this issue for a bit. I\'ll ping you around ' +
-      moment(props.unfreezeMoment).calendar() + ' :clock1: ' +
-      '<!-- ' + JSON.stringify(props) + '-->'
+      moment(props.unfreezeMoment).calendar() + ' :clock1: '
     }));
   }
 
-  unfreeze(issue, props) {
+  async checkUnfreeze(issue) {
+    const kv = metadata(this.github, issue, process.env.APP_ID);
+    const props = await kv.get('snooze');
```

Comment:
should the APP_ID that we pass in above do the equivalent of this for us? would snooze ever have access to any values that weren't `snooze`?

Reply:
The `APP_ID` just acts as a namespace to ensure apps won't override data for eachother. At the moment, `get` requires a key parameter, but I like the idea of modifying it to support being called without a key and just return all values.